src/richie/apps/courses/models/course.py

```diff
@@ -735,31 +742,34 @@ def compute_state(start, end, enrollment_start, enrollment_end):
         A static method not using the instance allows to call it with an Elasticsearch result.
         """
         if not start or not enrollment_start:
-            return CourseState(6)
+            return CourseState(CourseState.TO_BE_SCHEDULED)
```

Comment:
Also, just wanted to say this is much clearer than passing a number 👍

Reply:
:pray: